cfpb/cfgov-refresh
# Remove duplicated CF forms code

The enhancements code for cf-forms is no longer needed now that the
project is using Capital Framework v4

## Removals

- All of the styles in the cf-forms enhancements that already exist in Capital Framework

## Testing

1. Run `gulp build`, make sure it doesn't error
2. Navigate through the site and ensure the forms still look the same.

## Todos

- Move the rest of the enhancements to Capital Framework.

## Checklist
